pkgs/by-name/do/dooit/package.nix

```diff
@@ -64,8 +73,8 @@ python3.pkgs.buildPythonApplication rec {
 
   meta = with lib; {
     description = "TUI todo manager";
-    homepage = "https://github.com/kraanzu/dooit";
-    changelog = "https://github.com/kraanzu/dooit/blob/v${version}/CHANGELOG.md";
+    homepage = "https://github.com/${username}/${pname}";
+    changelog = "https://github.com/${username}/${pname}/blob/v${ver}/CHANGELOG.md";
```

Comment:
Would like to undo these. There's a lot of discussions around why it might be more DRY but is troublesome in nixpkgs.

Reply:
Hmm..didn't know, I'll undo it. Why is this troublesome though?

Reply:
Changing meta attributes based on package overrides, i believe.